R/clean_acs.R

```diff
@@ -37,7 +38,20 @@ clean_acs <- function(geography, variables, year, span, key, ...) {
            stco_name = NAME) %>%
     select_if(~sum(!is.na(.)) > 0)
   
+  set_label(output)<-names(output)
```

Comment:
set meaningful names as labels (in case names are changed later, can still tell what variables mean)